exchanges/hitbtc/hitbtc_types.go

```diff
@@ -370,3 +375,234 @@ type WsTrade struct {
+	ID            string        `json:"id"`
+	ClientOrderID string        `json:"clientOrderId"`
+	Symbol        currency.Pair `json:"symbol"`
+	Side          string        `json:"side"`
+	Status        string        `json:"status"`
+	Type          string        `json:"type"`
+	TimeInForce   string        `json:"timeInForce"`
+	Quantity      string        `json:"quantity"`
+	Price         string        `json:"price"`
+	CumQuantity   string        `json:"cumQuantity"`
+	PostOnly      bool          `json:"postOnly"`
+	CreatedAt     string        `json:"createdAt"`
+	UpdatedAt     string        `json:"updatedAt"`
+	ReportType    string        `json:"reportType"`
+}
+
+// WsReportResponse report response for auth subscription to reports
+type WsReportResponse struct {
+	Params WsReportResponseData `json:"params"`
+}
+
+// WsReportResponseData Report data for WsReportResponse
+type WsReportResponseData struct {
+	ID            string        `json:"id"`
+	ClientOrderID string        `json:"clientOrderId"`
+	Symbol        currency.Pair `json:"symbol"`
+	Side          string        `json:"side"`
+	Status        string        `json:"status"`
+	Type          string        `json:"type"`
+	TimeInForce   string        `json:"timeInForce"`
+	Quantity      string        `json:"quantity"`
+	Price         string        `json:"price"`
+	CumQuantity   string        `json:"cumQuantity"`
+	PostOnly      bool          `json:"postOnly"`
+	CreatedAt     string        `json:"createdAt"`
+	UpdatedAt     string        `json:"updatedAt"`
+	ReportType    string        `json:"reportType"`
+	TradeQuantity string        `json:"tradeQuantity"`
+	TradePrice    string        `json:"tradePrice"`
+	TradeID       int64         `json:"tradeId"`
+	TradeFee      string        `json:"tradeFee"`
+}
+
+// WsSubmitOrderRequest WS request
+type WsSubmitOrderRequest struct {
+	Method string                   `json:"method"`
+	Params WsSubmitOrderRequestData `json:"params"`
+	ID     int64                    `json:"id"`
+}
+
+// WsSubmitOrderRequestData WS request data
+type WsSubmitOrderRequestData struct {
+	ClientOrderID string        `json:"clientOrderId"`
+	Symbol        currency.Pair `json:"symbol"`
+	Side          string        `json:"side"`
+	Price         string        `json:"price"`
+	Quantity      string        `json:"quantity"`
+}
+
+// WsSubmitOrderSuccessResponse WS response
+type WsSubmitOrderSuccessResponse struct {
+	Result WsSubmitOrderSuccessResponseData `json:"result"`
+	ID     int64                            `json:"id"`
+}
+
+// WsSubmitOrderSuccessResponseData WS response data
+type WsSubmitOrderSuccessResponseData struct {
+	ID            string `json:"id"`
+	ClientOrderID string `json:"clientOrderId"`
+	Symbol        string `json:"symbol"`
+	Side          string `json:"side"`
+	Status        string `json:"status"`
+	Type          string `json:"type"`
+	TimeInForce   string `json:"timeInForce"`
+	Quantity      string `json:"quantity"`
```

Comment:
Quantity, Price and CumQuantity can be float64s
CreatedAt and UpdatedAt can be time.Time